doc/userguide/install.rst

```diff
@@ -205,8 +231,6 @@ To remove Suricata from your system::
 
     sudo apt-get remove suricata
 
-
-
```

Comment:
nit: nothing should change on these lines in this commit. No deletions, no additions

Reply:
Okay, noted